py3status/modules/sysdata.py

```diff
@@ -9,10 +9,8 @@
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
     mem_unit: the unit of memory to use in report, case insensitive.
         ['dynamic', 'KiB', 'MiB', 'GiB'] (default 'GiB')
-    padding: length of space padding to use on the left
-        (default 0)
-    precision: precision of values
-        (default 2)
+    temp_unit: unit used for measuring the temperature
+        (default '°C')
```

Comment:
It adds the option to show the unit.
I'll add the code to change the unit based on `temp_unit`.